teamC/JeongDayoung/.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL = "mysql://user:password@127.0.0.1:3306/westagram2"
+
+database_url = url;
```

데이터베이스 URL이 두 개네요?

무슨 차이가 있나요?

```suggestion
DATABASE_URL = "mysql://user:password@127.0.0.1:3306/westagram2"

database_url = url;
```

---

port를 잘못 적었습니다!